src/codec/framed_write.rs

```diff
@@ -197,6 +200,17 @@ where
                 Some(Next::Continuation(frame)) => {
                     // Buffer the continuation frame, then try to write again
                     if let Some(continuation) = frame.encode(&mut self.hpack, self.buf.get_mut()) {
+
+                        // We previously had a CONTINUATION, and after encoding
+                        // it, we got *another* one? Let's just double check
+                        // that at least some progress is being made...
+                        if self.buf.get_ref().len() == frame::HEADER_LEN {
+                            // If *only* the CONTINUATION frame header was
+                            // written, and *no* header fields, we're stuck
+                            // in a loop...
+                            panic!("CONTINUATION frame write loop; header value too big to encode");
```

Comment:
Since the expectation is that the check in `Send` should return an error before we ever hit this, I think this should probably be an `unreachable!` and/or make it clear that this is an internal implementation bug?

Reply:
>  make it clear that this is an internal implementation bug

It seems clear to me that it's a bug, I'm not 100% certain it's completely unreachable...